Scripts/tests/unit/test_logit.py

```diff
@@ -29,6 +29,8 @@ class Purpose:
         zi = numpy.array(INTERNAL_ZONES + EXTERNAL_ZONES)
         zd = ZoneData(ZONEDATA_PATH, zi, "uusimaa")
         zd["car_users"] = pandas.Series(0.5, zd.zone_numbers)
+        zd["cost"] = pandas.Series(0.0, index=zi)
+        zd["within_zone"] = pandas.Series(1, index=zi)
```

Comment:
Good that you added "cost". But I think this should without adding "within_zone".